src/locales/en/translation.json

```
@@ -1,18 +1,27 @@
 {
   "buttons": {
     "add_row": "Add row",
+    "back": "Back",
     "cancel": "Cancel",
     "close": "Close",
     "confirm": "Confirm",
     "delete": "Delete",
+    "done": "Done",
```

Biome (1.9.4)

[error] The key done was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)